src/SelectableBox/index.jsx

```diff
@@ -43,6 +43,7 @@ const SelectableBox = React.forwardRef(({
     hidden: inputHidden,
     ref: inputRef,
     tabIndex: -1,
+    'aria-label': 'checkbox',
```

Comment:
[clarification] Does this make sense to always be "checkbox" for all boxes in a `SelectableBox` set? E.g., will it need to be overridable by the consumer?